### DISCUSSION TITLE
Transfer Some Issue to Discussion?

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#494] [General]
Since we have enabled Discussion, should we transfer some discussion issues to the discussion panel?

**Comment by pc223** [comment]
What kind of issues that you think we should move to Discussion right now?

I think Discussion can use for a lot of things, even things that traditionally used the Issues page or Discord, like bug reports, feature requests, announcements, Q&A, ideas, feedback gathering,...

The question is what is the best approach for Flow? 😄

I lean towards we incrementally move 90% of discussions to the new Discussion, 10% use Issues/Pull request, Discord for backup 😸

**Reply by taooceros** [reply]
I think most issues can be moved to discussion, unless they are bugs and enhancement that will be implemented soon.

**Comment by pc223** [comment]
@Flow-Launcher/team ping for discussing 😄

**Comment by jjw24** [comment]
> I think Discussion can use for a lot of things, even things that traditionally used the Issues page or Discord, like bug reports, feature requests, announcements, Q&A, ideas, feedback gathering,...

yes makes sense, issue if it's actually confirmed one, everything else that needs discussion/convo can come here. However i think bug reports can still sit with issues.

I think we can move those non-issues when we come across them gradually

**Comment by pc223** [comment]
The conversion from issues to discussion already supported! I thought we have to deal with api and stuff

![image](https://user-images.githubusercontent.com/10551242/123591877-ba375280-d816-11eb-8f80-eeb6d2c27323.png)

The issue will be closed and redirect people to Discussion

![image](https://user-images.githubusercontent.com/10551242/123592211-21550700-d817-11eb-8e7b-c93663e2fcd3.png)

**Reply by jjw24** [reply]
Neat

**Reply by pc223** [reply]
Seems Github still have problems with this feature, It's rotating for like 5min now 😅 

![image](https://user-images.githubusercontent.com/10551242/123604157-c75b3e00-d824-11eb-9cb4-0c5ad56aff43.png)


Related:
https://github.com/github/feedback/discussions/2861

**Reply by taooceros** [reply]
Maybe you and me are converting the same issue XD, so something weird is happening.

**Reply by pc223** [reply]
> Maybe you and me are converting the same issue XD, so something weird is happening.

Not sure, no, I'm not converting, I just convert the Wiki issue like 15min ago, and It stuck 😝

**Reply by taooceros** [reply]
Isn't the wiki issue converted successfully 🤣

**Reply by pc223** [reply]
Nope, still rotating 😅 

https://github.com/Flow-Launcher/Flow.Launcher/discussions/504

**Reply by jjw24** [reply]
you guys break github lol 😅

**Reply by jjw24** [reply]
no wonder i am getting a million notifications from it 😝

**Reply by taooceros** [reply]
🤣🤣

**Reply by pc223** [reply]
I managed to fix it! The trick is to use the Github app on Android, so you can delete the discussion that "being migrated", and re-convert the issue again 😎

**Reply by jjw24** [reply]
lol

**Reply by jjw24** [reply]
good find

**Reply by pc223** [reply]
@jjw24  The conversion seems to create a bit of noise (locked issue instead of delete them), and UX problem too, beware users complaint: "Why did you close my issue????" when in fact it just transferred 😅 

Related:
https://github.com/github/feedback/discussions/3197

**Reply by jjw24** [reply]
Yeah I was a bit unsure what the end user experience would be like with closing and locking issue on convert. 

Our users are nice and understanding, we will survive :)

**Comment by jjw24** [comment]
i wonder what it looks like when you convert issue to discussion, then create issue from the discussion then again convert the issue to new discussion lol

**Reply by pc223** [reply]
🤣 

![image](https://user-images.githubusercontent.com/10551242/123633916-fe8d1780-d843-11eb-927c-0340862b49fe.png)
